sys/include/net/coap.h

```diff
@@ -138,7 +139,9 @@ extern "C" {
  */
```

Comment:
You could add an anchor to this group and reference it above:

```C
 /**
  * @name    Content-Format option codes
  * @anchor  net_coap_format
  * @{
  */
```

```
@deprecated  Deprecated in favour of [`COAP_FORMAT_`](@ref net_coap_format)
             style defines
```

Reply:
> You could add an anchor to this group and reference it above:

Awesome, I was looking for something like that!

Reply:
Done